apps/etterlatte-saksbehandling-ui/client/src/components/vurderingsboks/VurderingsboksWrapper.tsx

```diff
@@ -90,11 +95,9 @@ export const VurderingsboksWrapper = (props: Props) => {
               loading={lagrer}
               variant={'primary'}
               size={'small'}
-              onClick={async () => {
+              onClick={() => {
                 setLagrer(true)
-                new Promise((resolve) => resolve(props.lagreklikk(() => setRediger(false)))).finally(() =>
-                  setLagrer(false)
-                )
+                new Promise((resolve) => resolve(props.lagreklikk(reset))).catch(reset)
```

Comment:
Hvorfor er de to onClick-funksjonene forskjellige? Kunne disse bare vært definert der reset er definert?